@theme/markdoc/components.tsx

```diff
@@ -3,13 +3,13 @@ import { useLocation } from 'react-router-dom';
 // @ts-ignore
 import dynamicReact from '@markdoc/markdoc/dist/react';
 import { Link } from '@redocly/theme/components/Link/Link';
-import { useThemeHooks } from '@redocly/theme/core/hooks'
+import { useThemeHooks } from '@redocly/theme/core/hooks';
 import { idify } from '../helpers';
 import { Button } from '@redocly/theme/components/Button/Button';
 
 export {default as XRPLoader} from '../components/XRPLoader';
 export { XRPLCard, CardGrid } from '../components/XRPLCard';
-
+export * from '../components/Amendments';
```

Comment:
This is OK but I think the better practice might be to explicitly list the things that we're exporting, like this:

```suggestion
export { AmendmentsTable, AmendmentDisclaimer, Badge } from '../components/Amendments';
```